web/service/src/main/resources/logback.xml

```diff
@@ -6,8 +6,22 @@
             </pattern>
         </encoder>
     </appender>
+    <appender name="CONSOLE_JSON" class="ch.qos.logback.core.ConsoleAppender">
+       <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="com.google.cloud.spring.logging.StackdriverJsonLayout">
+                <includeTraceId>true</includeTraceId>
+                <includeSpanId>true</includeSpanId>
+                <includeLevel>true</includeLevel>
+                <includeThreadName>true</includeThreadName>
+                <includeMDC>true</includeMDC>
+                <includeLoggerName>true</includeLoggerName>
+            </layout>
+         </encoder>
+
+       </appender>
 
     <root level="INFO">
         <appender-ref ref="STDOUT" />
+        <appender-ref ref="CONSOLE_JSON" />
```

Comment:
The logs will be duplicated as there are two `ConsoleAppender` appenders. We approach this in two ways:
1. Detect when the Spring context is deployed in the cloud vs local. In the cloud, use the CONSOLE_JSON appended, and in local STDOUT.
2. Remove STDOUT in favor of CONSOLE_JSON